tests/ops/api/v1/endpoints/test_identity_verification_endpoints.py

```diff
@@ -0,0 +1,96 @@
+import pytest
+from starlette.testclient import TestClient
+
+from fidesops.ops.api.v1.scope_registry import EMAIL_READ, IDENTITY_VERIFICATION_READ
+from fidesops.ops.api.v1.urn_registry import ID_VERIFICATION_CONFIG, V1_URL_PREFIX
+from fidesops.ops.core.config import config
+
+
+class TestGetIdentityVerificationConfig:
+    @pytest.fixture(scope="function")
+    def url(self) -> str:
+        return V1_URL_PREFIX + ID_VERIFICATION_CONFIG
+
+    @pytest.fixture(scope="function")
+    def subject_identity_verification_required(self):
+        """Override autouse fixture to enable identity verification for tests"""
+        original_value = config.execution.subject_identity_verification_required
+        config.execution.subject_identity_verification_required = True
+        yield
+        config.execution.subject_identity_verification_required = original_value
```

Comment:
Since the fixture you are overriding if function scoped it is going to reset after the test anyways so you so don't have to do it here. That said it isn't hurting anything so it's fine it keep it as a double check.